assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +71,99 @@ angular.module('openshiftConsole')
       console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
     };
 
+    var podRelationships = function() {
+      $scope.servicePodsNoDeployment = {};
+      $scope.solitaryPods = {};
+
+      angular.forEach($scope.pods, function(pod, name){
+        var replicationcontrollers = [];
+        var services = [];
+        angular.forEach($scope.deployments, function(deployment, depName){
+          var ls = new LabelSelector(deployment.spec.selector);
+          if (ls.matches(pod)) {
+            replicationcontrollers.push(depName);
+          }
+        });
+        angular.forEach($scope.services, function(service, svcName){
+          var ls = new LabelSelector(service.spec.selector);
+          if (ls.matches(pod)) {
+            services.push(svcName);
+            if (replicationcontrollers.length == 0) {
+              $scope.servicePodsNoDeployment[svcName] = $scope.servicePodsNoDeployment[svcName] || {};
+              $scope.servicePodsNoDeployment[svcName][name] = pod;
+            }
+          }
+        });
+        if (replicationcontrollers.length == 0 && services.length == 0) {
+          $scope.solitaryPods[name] = pod;
+        }
+      });
+    };
+
+    var deploymentPods = function() {
+      $scope.deploymentPods = {};
+      angular.forEach($scope.deployments, function(deployment, name) {
+        var ls = new LabelSelector(deployment.spec.selector);
+        $scope.deploymentPods[name] = ls.select($scope.pods);
+      });
+
+      console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
+    };    
+
+    var serviceDeploymentConfigs = function() {
+      $scope.serviceDeploymentConfigs = {};
+      angular.forEach($scope.deploymentConfigs, function(deploymentConfig, depName){
+        if (!deploymentConfig) {
+          // Handle the "special" null deployment config, do we still even need this thing
+          return;
+        }
+        var foundMatch = false;
+        angular.forEach($scope.services, function(service, name){
+          $scope.serviceDeploymentConfigs[name] = $scope.serviceDeploymentConfigs[name] || {};
+
+          // TODO this should be checking if the service selector covers the
+          // deployment config selector instead of checking equality
+          if (service.spec.selector.name == deploymentConfig.template.controllerTemplate.replicaSelector.name) {
```

Comment:
Note the TODO, which i forgot about.  It does not always have a name key, this was temporary to get some testing done, it will have to be fixed in this PR to actually check that service selector covers the replica selector.  I am thinking this should be a method on a LabelSelector object to check if a LabelSelector covers another LabelSelector